index.html

```diff
@@ -13,5 +13,8 @@ <h1>Check out this cool frog!</h1>
         <img src="https://external-preview.redd.it/aoOxi1firITL5t_svWtPjGSY0jgGgaLXibDcogLJX8Q.jpg?auto=webp&s=652e931cf9f51c2dcf16278de318d5fd2b877875" alt="frog with umbrella">
         <h2>I like this frog.</h2>
     </main>
+    <div>
```

Comment:
Footer looks good, but let's make sure to use a semantic `<footer>` element.

Reply:
Oh yeah! Great suggestion, you're so smart!